src/commands/registerCommands.ts

```diff
@@ -61,6 +61,7 @@ import { logInToDockerCli } from "./registries/logInToDockerCli";
 import { logOutOfDockerCli } from "./registries/logOutOfDockerCli";
 import { pullImage, pullRepository } from "./registries/pullImages";
 import { reconnectRegistry } from "./registries/reconnectRegistry";
+import { reportIssue } from "./reportissue";
```

Comment:
```suggestion
import { reportIssue } from "./reportIssue";
```
Linux filesystem is case-sensitive so this might prevent building there.